tests/Seeds.ts

```diff
@@ -318,14 +318,25 @@ async function seed(): Promise<void> {
   const MERCH_ITEM_1 = MerchFactory.fakeItem({
     collection: MERCH_COLLECTION_1,
     itemName: 'Unisex Hack School Anorak',
-    picture: 'https://i.imgur.com/jkTcUJO.jpg',
     description: 'San Diego has an average annual precipitation less than 12 inches,'
     + 'but that doesn\'t mean you don\'t need one of these.',
     monthlyLimit: 1,
     lifetimeLimit: 1,
     hidden: false,
     hasVariantsEnabled: true,
   });
+  const MERCH_ITEM_1_PHOTO_0 = MerchFactory.fakePhoto({
+    merchItem: MERCH_ITEM_1,
+  });
+  const MERCH_ITEM_1_PHOTO_1 = MerchFactory.fakePhoto({
+    merchItem: MERCH_ITEM_1,
+    position: 1,
+  });
+  const MERCH_ITEM_1_PHOTO_2 = MerchFactory.fakePhoto({
+    merchItem: MERCH_ITEM_1,
+    picture: 'https://i.imgur.com/pSZ921P.png',
+    position: 2,
+  });
```

Comment:
nit: curious why only the last photo has a picture, but not the first two?

Reply:
And why photo_0 doesn't have a position?

Reply:
Fake photo actually already generates a fake url for the object, as well as the default position. I kinda just assign different values when necessary.

Reply:
Perhaps for consistency sake I should make everything explicit.